ftw/mail/configure.zcml

```diff
@@ -42,6 +42,12 @@
         permission="zope2.View"
         />
 
+    <!-- Mail object creator -->
+    <adapter
+        for="*
+             zope.publisher.interfaces.browser.IBrowserRequest"
```

Comment:
@deiferni Does adapting the request actually make sense? The inbound script is usually called from a script, which has no request?
How is do you use a more specific adapter?

Like:

```python

...
alsoProvides(getGlobalRequest(), IRequestLayer)
...

``` 

This means you have to implement your own inbound script if you want to use a more specific adapter.